CapacitorCommunityIntercom.podspec

```diff
@@ -9,10 +9,10 @@ Pod::Spec.new do |s|
   s.license = package['license']
   s.homepage = package['repository']['url']
   s.author = package['author']
-  s.source = { git: package['repository']['url'], tag: s.version.to_s }
-  s.source_files = 'ios/Plugin/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
-  s.ios.deployment_target  = '11.0'
-  s.static_framework = true
+  s.source = { :git => package['repository']['url'], :tag => s.version.to_s }
+  s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
+  s.ios.deployment_target  = '12.0'
   s.dependency 'Capacitor'
```

Comment:
perhaps lock this to `s.dependency 'Capacitor', '~>3' ? `

Reply:
@atomassoni done!